dev/ci/README.md

```diff
@@ -36,10 +36,6 @@ On the condition that:
 - You do not push, to the branches that we test, commits that haven't been
   first tested to compile with the corresponding branch(es) of Coq.
 
-- Your development compiles in less than 35 minutes with just two threads.
-  If this is not the case, consider adding a "lite" target that compiles just
-  part of it.
-
 In case you forget to comply with these last three conditions, we would reach
```

Comment:
I'd suggest we keep a paragraph about build time, even if we don't put a concrete number; "reasonable time" should be mentioned, there are a few Coq developments that may take in the order of > 10 hours.

Reply:
Sounds good.